projects/common/src/time/time-range.service.ts

```diff
@@ -66,7 +66,7 @@ export class TimeRangeService {
     this.setTimeRange(this.getCurrentTimeRange());
   }
 
-  private globalTimeRangeInitialization(): Observable<TimeRange> {
+  private globalTimeRangeInitConfigAndChanges(): Observable<TimeRange> {
```

Comment:
nit: not changes - we're only take(1)'ing. Maybe `getInitialTimeRange()` and below `getPageTimeRangeChanges()`